### PR TITLE
imrpovement: cleanup namespaces

### PR DESCRIPTION
**Pull request by Shabirmean**
### Description
- Post merge, the namespaces and resources created for PRs are not deleted
- This causes a resource limitation
- So we want to make sure when PRs are merged the namespaces created for them are removed

### Changes
- Update the Cloudbuild trigger that runs on pushes to main to check and delete the most recently closed PR namespaces

### CODE REVIEW

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**34.136.112.245**](http://34.136.112.245)
- **Embedded H2** DB backed deployment: [**34.122.197.1**](http://34.122.197.1)

**Review by ckim328 — APPROVED** [review]
LGTM, just left a small point of clarification, but the cloudbuild steps look good to me!

File: .github/cloudbuild/pos-deploy-release.yaml [hunk 1]
```diff
@@ -33,7 +33,6 @@ steps:
     gcloud container clusters get-credentials --zone $_MAIN_CLUSTER_ZONE $_MAIN_CLUSTER
     kubectl create namespace $_MAIN_NS-db
     kubectl create namespace $_MAIN_NS-inmemory
-    kubectl apply -f /workspace/namespace.yaml
```
Comment (ckim328) [inline .github/cloudbuild/pos-deploy-release.yaml:36]: point of clarification - you're taking this out since the ns will already exist in this cluster?

Comment (Shabirmean) [reply .github/cloudbuild/pos-deploy-release.yaml:36]: This was an error from a previous cleanup I did. Previously this whole step was broken into two:
- Create the `/workspace/namespace.yaml` file
- Then apply it on the next cloudbuild step
Then I realized I could just do it all at once like we see in line 34,35 above. After adding those two lines I forgot to delete this line. I noticed it when I was making changes to this PR.

**Comment by Shabirmean** [comment]
> LGTM, just left a small point of clarification, but the cloudbuild steps look good to me!

Thank you very much for the speedy reviews 🙏🏼

File: .github/cloudbuild/pos-deploy-release.yaml (hunk 1 above)
Comment (ckim328) [reply .github/cloudbuild/pos-deploy-release.yaml:36]: lol my eyes just glazed over. sounds good and makes sense to me!

**Pull request merged**